sdk/storage/Azure.Storage.Common/src/ParallelTransferOptions.cs

```diff
@@ -19,7 +19,7 @@ public struct ParallelTransferOptions : IEquatable<ParallelTransferOptions>
         /// <summary>
         /// The maximum number of threads that may be used in a parallel transfer.
         /// </summary>
-        public int? MaximumThreadCount { get; set; }
```

Comment:
Nit - can we change the comment from "maximum number of threads" to "maximum number of workers"?  It's not always threads, etc.